prime-router/metadata/schemas/covid-19.schema

```diff
@@ -775,6 +775,13 @@ elements:
     hl7Field: OBX-7
     documentation: The reference range of the lab result, such as “Negative” or “Normal”. For IgG, IgM and CT results that provide a value you MUST fill out this filed.
 
+  - name: reporting_facility
+    type: HD
+    hl7Field: MSH-4
+    documentation: |
+      The reporting facility for the message, as specified by the receiver. This is typically used if PRIME is the
+      aggregator
+
```

Comment:
For Ohio, and presumably others in the future, we might need to pass in the aggregator information into the MSH header. That could be done here.